src/main/kotlin/Main.kt

```diff
@@ -1,54 +1,102 @@
 import io.appwrite.Client
-import com.google.gson.GsonBuilder
-import com.google.gson.JsonElement
-import com.google.gson.JsonParser
 import io.appwrite.exceptions.AppwriteException
 import io.appwrite.services.Database
 import io.appwrite.services.Functions
 import io.appwrite.services.Storage
 import io.appwrite.services.Users
-import kotlinx.coroutines.runBlocking
 import java.io.File
-import kotlin.math.min
-
 
 val client = Client()
-    .setEndpoint("https://localhost/v1")
-    .setProject("YOUR PROJECT ID")
-    .setKey("YOUR API KEY")
```

Comment:
Maybe we should leave these as they are? to let the developer know they have to fill values in ?